src/index.ts

```
@@ -103,6 +104,59 @@
             path = args.path.replace(/^\/npm\//, "/");
           }
 
+          console.log("sjlkdljasljkasd", args.path);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)